.github/workflows/ci_cd.yml

```diff
@@ -177,8 +177,8 @@ jobs:
 
       - name: Scrape the stable documentation to PyMeilisearch
         run: |
-          VERSION=$(python -c "from ${{ env.PACKAGE_NAME }} import __version__; print('.'.join(__version__.split('.')[:2]))")
-          VERSION_MEILI=$(python -c "from ${{ env.PACKAGE_NAME }} import __version__; print('-'.join(__version__.split('.')[:2]))")
+          VERSION=$(python -c "from ansys_sphinx_theme import __version__; print('.'.join(__version__.split('.')[:2]))")
+          VERSION_MEILI=$(python -c "from ansys_sphinx_theme import __version__; print('-'.join(__version__.split('.')[:2]))")
           echo "Calculated VERSION: $VERSION"
```

Comment:
Oh... we changed the namespace?! This is going to have breaking changes for sure on our repos, on the way we import things.

Reply:
> Oh... we changed the namespace?! This is going to have breaking changes for sure on our repos, on the way we import things.

no we didnt change the name space, the ${{ env.PACKAGE_NAME }} is ansys-sphinx-theme which is causing the syntax error